env/env.js

```diff
@@ -139,6 +139,7 @@ const test = removeNullValues({
   ...base,
   FIRESTORE_URL: 'http://localhost:8081',  // for e2e tests with firestore emulator
   NODE_ENV: 'test',
+  REACT_APP_ENVIRONMENT: 'test',
```

Comment:
Needed to make `yarn client gulp test` work

Reply:
What issue are you seeing?

Reply:
`REACT_APP_ENVIRONMENT` is not defined, then `App.test.tsx` will fail the test